sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueAsyncClient.java

```diff
@@ -548,26 +548,28 @@ public URL getQueueUrl() {
      * @param maxMessages Optional. Maximum number of messages to get, if there are less messages exist in the queue than requested
      * all the messages will be returned. If left empty only 1 message will be retrieved, the allowed range is 1 to 32
      * messages.
-     * @param visibilityTimeout Optional. The timeout period for how long the message is invisible in the queue in seconds.
+     * @param visibilityTimeout Optional. The timeout period for how long the message is invisible in the queue.
      * If left empty the dequeued messages will be invisible for 30 seconds. The timeout must be between 1 second and 7 days.
+     * @param timeout An optional timeout to be applied to the network asynchronous operations.
      * @return Up to {@code maxMessages} {@link DequeuedMessage DequeuedMessages} from the queue. Each DeqeuedMessage contains
      * {@link DequeuedMessage#messageId() messageId} and {@link DequeuedMessage#popReceipt() popReceipt} used to interact
      * with the message and other metadata about the message.
      * @throws StorageException If the queue doesn't exist or {@code maxMessages} or {@code visibilityTimeout} is
      * outside of the allowed bounds
      */
-    public PagedFlux<DequeuedMessage> dequeueMessages(Integer maxMessages, Duration visibilityTimeout) {
+    public PagedFlux<DequeuedMessage> dequeueMessagesWithOptionalTimeout(Integer maxMessages, Duration visibilityTimeout,
```

Comment:
This is the async client, shouldn't have timeout in the public API, additionally let's remove the WithOptionalTimeout post fix that was added to the method name.